test/integration/infra.go

```diff
@@ -190,7 +190,7 @@ func (infra *infra) setup() error {
 		}
 	}
 	if infra.Ingress {
-		if err := deploy("ingress-proxy.yaml.tmpl", infra.IstioNamespace); err != nil {
+		if err := deploy("ingress-proxy.yaml.tmpl", infra.Namespace); err != nil {
```

Comment:
ingress proxy should be in istio namespace.

Reply:
The purpose of this change is to allow multiple runs, this is why ingress and egress have been moved back to the app namespace. When we will have isolated cluster, they can be run in IstioNamespace.